c++/src/kj/exception.h

```diff
@@ -369,7 +369,7 @@ KJ_NOINLINE ArrayPtr<void* const> getStackTrace(ArrayPtr<void*> space, uint igno
 // is never exactly equal to `space.begin()` due to this effect, even if `ignoreCount` is zero
 // since `getStackTrace()` needs to ignore its own internal frames).
 
-String stringifyStackTrace(ArrayPtr<void* const>);
+String stringifyStackTrace(ArrayPtr<void* const>) __attribute__((weak));
```

Comment:
Hmm, shouldn't the attribute go on the implementation, not the prototype? Otherwise how does the compiler know which implementation is the weak one?